hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneConfigUtil.java

```diff
@@ -124,4 +128,35 @@ public static ReplicationConfig resolveReplicationConfigPreference(
     }
     return replicationConfig;
   }
+
+  /**
+   * Limits or cap the client config value to server supported config value,
+   * if client config value crosses the server supported config value.
+   * @param clientConfValue - the client config value to be capped
+   * @param clientConfName - the client config name
+   * @param serverConfName - the server config name
+   * @param serverConfValue - the server config value
+   * @return the capped config value
+   */
+  public static long limitValue(long clientConfValue, String clientConfName,
+                                String serverConfName, long serverConfValue) {
+    long limitVal = clientConfValue;
+    if (clientConfValue > serverConfValue) {
+      LOG.warn("{} config value is greater than server config {} " +
```

Comment:
A badly configured client can flood the server logs, it is ok for the server to cap the limit and not warn any one. If any thing we can collect a metric for how many times this is occuring.

Reply:
@kerneltime , thank you for reviewing the code. I have handled to make it debug and have created a new JIRA for adding a metric for occurence of such case.

Reply:
Added JIRA - [HDDS-7439](https://issues.apache.org/jira/browse/HDDS-7439)